css/style.css

```diff
@@ -0,0 +1,180 @@
+/* * {
```

Comment:
Appearance matches the image:  The layout and general "feeling" of the site match with the wireframe.  I noticed two differences:  1 )"Some Call to Action" link not looking like a button / having a border around it.  2) Text in the "Why?" section could use a some spacing to move the words a bit further away from the images.  Everything else is right on and the flex-wrapping of the pictures is super great!